src/tests/ftest/dfuse/daos_build.yaml

```diff
@@ -3,8 +3,8 @@ hosts:
     - server-A
   test_clients:
     - client-C
-# Test typically takes around 15 minutes, set timeout for 30.
-timeout: 1800
+# Limit the test at four hours.
+timeout: 14400
```

Comment:
Can we afford to add a 4h test to the daily test set?

Reply:
It's currently two hours although probably on two distributions, and part of adding this test is so that we know how long it takes.  The main things are to be able to call it for PRs, to have a reference level and to be able to identify regressions so weekly would probably be acceptable.

Reply:
We had an unexplained test failure in this PR overnight although I did do a master merge yesterday.  One advantage of doing daily runs initially would be to get confidence in the test itself (it does significantly increase coverage), in writing this test I've found ~5 issues where the code just doesn't work so it's not unreasonable to assume that it'll expose some race conditions as well, if this test were weekly we'd only really see them manifest in PRs which isn't ideal.